src/TemplatesBuilder.js

```diff
@@ -67,6 +68,25 @@ export default class TemplatesBuilder {
     }
   }
 
+  buildClasses() {
```

Comment:
Need the optional classes and the default class.

Reply:
should this be a code comment?